site/guides/README.md

```diff
@@ -5,3 +5,6 @@
 - [Namespace provisioning UI](guides/namespace-provisioning-ui.md)
 - [Porch Installation Guide](guides/porch-installation.md)
 - [Porch User Guide](guides/porch-user-guide.md)
+- [Tenant Onboarding Guide](guides/tenant-onboarding.md)
+- [Variant Constructor Pattern](guides/variant-constructor-pattern.md)
+- [Value Propagation Pattern](guides/value-propagation.md)
```

Comment:
Just realized, you need to update `scripts/generate_site_sidebar/sidebar_template.md.tmpl` and run site-run-server to update the sidebar. You can also check them out locally.

Refer to the contributing guide for docs.
https://github.com/GoogleContainerTools/kpt/blob/main/CONTRIBUTING.md#documentation

Reply:
Thanks! Will do